newIDE/electron-app/app/ModalWindow.js

```diff
@@ -38,13 +38,19 @@ const loadModalWindow = ({
     center: true,
     webPreferences: {
       webSecurity: false,
+      enableRemoteModule: true,
```

Comment:
Not sure if this is still need with the latest Electron version(s)?

Reply:
Right, this isn't needed anymore. Removed now.